### PR TITLE
User2story

### PR DESCRIPTION
**Pull request by LizaMarie0421**
@GattieTime we added to the quiz model, edit-quiz.component.ts/.html, add-quiz.component.ts/.html, quiz-page.component.html

### CODE REVIEW

File: rss-client/src/app/Quiz/components/quiz-page/quiz-page.component.ts [hunk 1]
```diff
@@ -35,6 +35,7 @@ export class QuizPageComponent implements OnInit {
   ) {}
 
   ngOnInit(): void {
+
```
Comment (GattieTime) [inline rss-client/src/app/Quiz/components/quiz-page/quiz-page.component.ts:38]: Please do not submit empty line changes for a pull request.

File: rss-client/src/app/Quiz/components/quiz-page/quiz-page.component.ts [hunk 2]
```diff
@@ -49,7 +50,9 @@ export class QuizPageComponent implements OnInit {
         this.quizzesTaken = res;
       }
     });
-    this.quizservice.getAllQuizzes().subscribe((res) => (this.quizData = res));
```
Comment (GattieTime) [inline rss-client/src/app/Quiz/components/quiz-page/quiz-page.component.ts:52]: Please do not make cosmetic changes in a PR.

**Review by GattieTime — CHANGES_REQUESTED** [review]

File: rss-client/src/app/Quiz/admin/add-quiz/add-quiz.component.ts [hunk 3]
```diff
@@ -131,23 +173,33 @@ export class AddQuizComponent implements OnInit {
...
+            // sets quizTotalPoints to availablePoints (which is all the added questions points added together)
+            this.focusedQuiz.quizTotalPoints = this.focusedQuiz.availablePoints;
           } else if (result.type == 'delete') {
-            //TODO:remove question from database here
             this.focusedQuiz.questions = this.focusedQuiz.questions.filter(
               (x) => x.questionId != result.value.questionId
             );
             this.updateTotal();
             this.validate();
+            //Group 2 change
```
[the remark this answers is not in the document]
Comment (LizaMarie0421) [reply rss-client/src/app/Quiz/admin/add-quiz/add-quiz.component.ts:185]: yes we are keeping track!!

**Review by GattieTime — APPROVED** [review]
Looks good!

**Pull request merged**